lightblue/src/main/java/org/esbtools/eventhandler/lightblue/FindRequests.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright 2015 esbtools Contributors and/or its affiliates.
+ *
+ *  This file is part of esbtools.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.esbtools.eventhandler.lightblue;
+
+import com.redhat.lightblue.client.Literal;
+import com.redhat.lightblue.client.Projection;
+import com.redhat.lightblue.client.Query;
+import com.redhat.lightblue.client.Sort;
+import com.redhat.lightblue.client.request.data.DataFindRequest;
+
+import org.esbtools.eventhandler.lightblue.model.DocumentEventEntity;
+
+public abstract class FindRequests {
```

Comment:
At the risk of sounding like a cliche, I think we could move these classes into a util package... thoughts? Either way, I'd like to see them outside of the general package (related, I generally thing that package could be split up a bit). Thoughts?